codalab_service.py

```diff
@@ -326,7 +326,8 @@ def has_callable_default(self):
         name='worker_manager_worker_checkin_frequency_seconds',
         help='Number of seconds to wait between check-ins for a worker of the worker manager',
         type=int,
-        default=20,
+        # TODO(Ashwin): Change this back to 20 once we get websockets working with kubernetes workers.
+        default=5,
```

Comment:
cc @AndrewJGaut @percyliang

Reply:
Percy asked to keep this at 5 regardless, right? Just so we still have frequent checkins?